api/api/migrations/0058_userpreferences.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2.10 on 2024-03-13 21:24
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def create_user_preferences(apps, schema_editor):
+    UserPreferences = apps.get_model("api", "UserPreferences")
+    User = apps.get_model("auth", "User")
+    for user in User.objects.all():
+        if not hasattr(user, "userpreferences"):
+            UserPreferences.objects.create(user=user)
+            user.userpreferences.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('api', '0057_alter_sensitiveaudio_options'),
```

Comment:
This change, combined with the rename of this migration file, might avoid the need for rebasing the PR.
```suggestion
        ('api', '0058_moderation_decision'),
```

Reply:
Please rebase the PR rather than trying to avoid it! There's no reason to avoid rebasing and it's the clearest and most consistent way to avoid shenanegans with migration files, which we should prioritise avoiding.

Reply:
Sorry about that. I misspoke (fixed now). I suggested the change to avoid having to delete and remake the migration, rebasing the PR is absolutely recommended to make sure everything works as expected against the current state of `main`.

Reply:
Updated in 28f8b6b2d02ca529c6e8cb63082252f42146e838